### DISCUSSION TITLE
Spring Security configuration

### DISCUSSION THREAD

**Discussion by Michael-M-93** [community/community#135493] [Programming Help]
### Body

Hello,

I am looking for spring security code with jwt token.
I am still learning and i dont know how started with security config.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rlNkoo** [comment]
Below you can find spring-security-configuration repository (last version)
Inculed implementation of jwt token and user roles.

https://github.com/rlNkoo/spring-security-configuration

**Reply by Michael-M-93** [reply]
Thank you sir!

**Comment by ebndev** [comment]
Hi @Michael-M-93 & @rlNkoo, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
